sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -652,7 +651,7 @@ def dead_letter_reason(self):
         return None
 
     @property
-    def dead_letter_source(self):
+    def _dead_letter_source(self):
```

Comment:
these three should be moved from `PeekedMessage` to `ReceivedMessage`.

Reply:
@rakshith91 , actually I think you could just revert the change, as my settlement move will completely remove the `PeekedMessage` class https://github.com/Azure/azure-sdk-for-python/pull/14681